src/Umbraco.Cms.Integrations.Crm.ActiveCampaign/Models/Dtos/ResponseDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Umbraco.Cms.Integrations.Crm.ActiveCampaign.Models.Dtos
+{
+    public class ResponseDto
```

Comment:
Am thinking this seems a little odd having both the single and collection of forms response mapped to a single object.  Would it be better to have `FormResponseDto` and `FormCollectionResponseDto`, and map to the appropriate one

`FormCollectionResponseDto` would have a single property which would be a collection of `FormDto`.

`FormResponseDto` would have a single `FormDto`.

Both could inherit from a base type that defines `Message`.